test/freetype_test.py

```diff
@@ -1795,5 +1795,70 @@ def test_get_error(self):
         self.assertIsNone(error_msg)
 
 
+class VisualTests(unittest.TestCase):
+
+    __tags__ = ["interactive"]
+
+    screen = None
+    aborted = False
+
+    def setUp(self):
+        if self.screen is None:
+            pygame.init()
+            self.screen = pygame.display.set_mode((600, 200))
+            self.screen.fill((255, 255, 255))
+            pygame.display.flip()
+            self.f = pygame.freetype.Font(None)
+
+    def tearDown(self):
+        pygame.quit()
+
+    def query(self, clip_rect, is_clipped=False):
+        self.screen.fill((255, 255, 255))
```

Comment:
the screen background needs to be a different colour to the text that is supposed to be clipped
```suggestion
        self.screen.fill((128, 128, 128))
```